src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -206,11 +205,6 @@ def get(self, request: Request, project: Project) -> Response:
                     # Later on, if there are matching code mappings this will be overwritten
                     result["error"] = "stack_root_mismatch"
                     continue
-                if (
-                    filepath.startswith(config.stack_root)
```

Comment:
Matching the stack root is not anymore a required check. It was when this code was first added.